LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2014 Grant Goodyear
```

Comment:
I also don't care (not that I've done much), and considering both versions were based on the Rnoweb (which as part of VIM is under a GPL-compatible charityware license), I'm not sure how legitimate the MIT license is anyway. I just left it as it was.